src/infer_response.h

```diff
@@ -246,8 +263,14 @@ class InferenceResponse {
       const ResponseAllocator* allocator, void* alloc_userp,
       TRITONSERVER_InferenceResponseCompleteFn_t response_fn,
       void* response_userp,
-      const std::function<void(
-          std::unique_ptr<InferenceResponse>&&, const uint32_t)>& delegator);
+      const std::function<
+          void(std::unique_ptr<InferenceResponse>&&, const uint32_t)>& delegator
+#ifdef TRITON_ENABLE_METRICS
+      ,
+      std::shared_ptr<std::atomic<uint64_t>> responses_sent_,
```

Comment:
```suggestion
      std::shared_ptr<std::atomic<uint64_t>> responses_sent,
```
function arg here shouldn't have _ suffix in our current style - only member variables

Reply:
Fixed